snap/snapcraft.yaml

```diff
@@ -393,6 +400,10 @@ slots:
     interface: dbus
     name: org.freedesktop.portal.IBus
     bus: session
+  dbus-colord:
+    interface: dbus
+    name: org.freedesktop.ColorManager
+    bus: session
```

Comment:
According to d-feet, org.freedesktop.ColorManager is a system bus service; the one in the session (with the same binary) is org.freedesktop.ColorHelper